utils/docker-utils.sh

```diff
@@ -23,11 +23,34 @@ docker::get_current_service_status() {
 # - $1 : service name (eg. analytics-datastore-elastic-search)
 #
 docker::get_service_unique_errors() {
-    local -r SERVICE_NAME=${1:?"FATAL: await_container_startup parameter not provided"}
+    local -r SERVICE_NAME=${1:?"FATAL: get_service_unique_errors parameter not provided"}
 
     # Get unique error messages using sort -u
     docker service ps instant_"${SERVICE_NAME}" --no-trunc --format '{{ .Error }}' 2>&1 | sort -u
 }
+
+# Warn when the status is Preparing for 3 minutes
+#
+# Arguments:
+# - $1 : service name (eg. analytics-datastore-elastic-search)
+#
+docker::warn_preparing_status() {
```

Comment:
I think this isn't relevant anymore since y're pull images in the begining.

Reply:
cool done